frontend/test/setupTests.js

```diff
@@ -0,0 +1,3 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter(), disableLifecycleMethods: true });
```

Comment:
What does this do?

Reply:
(Maybe you can add one line of documentation here)

Reply:
I removed these as the testing is not implemented properly.